update.go

```diff
@@ -60,6 +60,9 @@ func updateCLI(channel string) {
 	if Autoupdate == "no" {
 		return
 	}
+	if config.LockChannel != "" {
+		channel = config.LockChannel
```

Comment:
@dickeyxxx need to throw an error here if they try and do `heroku update foo` and `LockChannel` != `foo`